### ISSUE TITLE
Gradle 6.5: A lot of deprecation mesages

### ISSUE THREAD

**Issue by remal** [hierynomus/license-gradle-plugin#186]
I'm using `0.15.0` version of `com.github.hierynomus.license` plugin and getting a lot of deprecation messages:
```
Property 'check' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0. See https://docs.gradle.org/6.5/userguide/more_about_tasks.html#sec:up_to_date_checks for more details.
Property 'dryRun' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0. See https://docs.gradle.org/6.5/userguide/more_about_tasks.html#sec:up_to_date_checks for more details.
Property 'encoding' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0. See https://docs.gradle.org/6.5/userguide/more_about_tasks.html#sec:up_to_date_checks for more details.
Property 'headerDefinitions' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0. See https://docs.gradle.org/6.5/userguide/more_about_tasks.html#sec:up_to_date_checks for more details.
Property 'inheritedMappings' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0. See https://docs.gradle.org/6.5/userguide/more_about_tasks.html#sec:up_to_date_checks for more details.
Property 'inheritedProperties' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0. See https://docs.gradle.org/6.5/userguide/more_about_tasks.html#sec:up_to_date_checks for more details.
Property 'internalMappings' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0. See https://docs.gradle.org/6.5/userguide/more_about_tasks.html#sec:up_to_date_checks for more details.
Property 'skipExistingHeaders' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0. See https://docs.gradle.org/6.5/userguide/more_about_tasks.html#sec:up_to_date_checks for more details.
Property 'strictCheck' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0. See https://docs.gradle.org/6.5/userguide/more_about_tasks.html#sec:up_to_date_checks for more details.
Property 'URI' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0. See https://docs.gradle.org/6.5/userguide/more_about_tasks.html#sec:up_to_date_checks for more details.
Property 'useDefaultMappings' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0. See https://docs.gradle.org/6.5/userguide/more_about_tasks.html#sec:up_to_date_checks for more details.
Property 'validHeaders' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0. See https://docs.gradle.org/6.5/userguide/more_about_tasks.html#sec:up_to_date_checks for more details.
```

Could you please appropriate annotations on the properties?

**Comment by aalmiray** [comment]
Duplicate of #179

**Comment by tbroyer** [comment]
There are other deprecation messages, related to the `downloadLicenses` task:
```
Property 'html' has redundant getters: 'getHtml()' and 'isHtml()'. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'ignoreFatalParseErrors' has redundant getters: 'getIgnoreFatalParseErrors()' and 'isIgnoreFatalParseErrors()'. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'includeProjectDependencies' has redundant getters: 'getIncludeProjectDependencies()' and 'isIncludeProjectDependencies()'. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'json' has redundant getters: 'getJson()' and 'isJson()'. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'reportByDependency' has redundant getters: 'getReportByDependency()' and 'isReportByDependency()'. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'reportByLicenseType' has redundant getters: 'getReportByLicenseType()' and 'isReportByLicenseType()'. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'xml' has redundant getters: 'getXml()' and 'isXml()'. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
```

**Comment by hierynomus** [comment]
This should be fixed with the newly release 0.16.1!

**Comment by andrii-ts** [comment]
Is 0.16.1 available for usage? 
> Plugin [id: 'com.github.hierynomus.license', version: '0.16.1'] was not found in any of the following sources
